server/server.js

```diff
@@ -13,76 +13,163 @@
 	// #region Imports
 	// NPM
 	const chalk = require("chalk");
+	const errorColor = chalk.red;
```

Comment:
Defining variables in the imports section. This should be done after all fo the imports